src/app/legacy/containers/StoryPromo/utilities/index.js

```diff
@@ -14,11 +11,7 @@ export const isMap = item => {
   return isCpsTypeMap || Boolean(hasMedia) || isOptimoMediaPromo;
 };
 
-export const getHeadingTagOverride = ({ pageType, isContentTypeGuide }) => {
-  if (pageType === MOST_WATCHED_PAGE) {
-    return 'h2';
-  }
-
+export const getHeadingTagOverride = ({ isContentTypeGuide }) => {
```

Comment:
Hopefully editors will no longer put links to most watched in story promos (if this is what that is), as they will lonk to 404s but also they will not have the right heading or a11y attributes anymore.